lib/active_merchant/billing/gateways/payeezy.rb

```diff
@@ -119,27 +127,34 @@ def add_authorization_info(params, authorization)
         params[:method] = method
       end
 
+      def add_creditcard_for_tokenization(params, payment_method, options)
+        params[:apikey] = options[:apikey]
```

Comment:
I think we can use `@options[:apikey]` here since it's passed in via `initialize`. It looks like the call to `super` will create this instance variable: https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateway.rb#L183.

Reply:
👍